docs/source/common_issues.rst

```diff
@@ -270,7 +269,8 @@ unexpected errors when combined with type inference. For example:
 
    lst = [A(), A()]  # Inferred type is List[A]
    new_lst = [B(), B()]  # inferred type is List[B]
-   lst = new_lst  # mypy will complain about this, because List is invariant
+   # mypy will complain about this, because List is invariant
+   lst = new_lst  # error: Incompatible types in assignment (expression has type "List[B]", variable has type "List[A]")
```

Comment:
Please advise whether keeping the explanation comments is useful, and whether the selected format is acceptable:
`<code>  # original explanation` becomes

```
# original explanation
<code>  # mypy error
```

Reply:
This looks fine to me. Though here the explanation is pretty informal, and it may be better to be more explicit about what this refers to. Here is one idea:

```
    # Mypy rejects the following assignment, because List is invariant
    lst = new_lst  # error: ...
```
```